sormas-backend/src/main/java/de/symeda/sormas/backend/user/User.java

```diff
@@ -234,6 +235,12 @@ public JurisdictionLevel getJurisdictionLevel() {
 		return jurisdictionLevel;
 	}
 
+	@Transient
```

Comment:
Could we remove this from the Entity and keep it in the UserProvider to avoid duplication?

Reply:
UserProvider is in Sormas-UI project. However, current implementation is on Sormas-Backend side. It means, that current implementation cannot be done in than way.. 
Doing vice-versa -  for accessing it from Sorma-UI and calling Sormas-Backend (but it still can require some more changes). Might be if any other module will require it, at that moment, this code can be refactored.